bitbots_splines/include/bitbots_splines/abstract_visualizer.h

```diff
@@ -10,6 +10,7 @@ namespace bitbots_splines {
 
 class AbstractVisualizer {
  protected:
+  int id = 0;
```

Comment:
It is used `abstract_visualizer.h:65` and `abstract_visualizer.h:105`, `abstract_visualizer.h:121` and `abstract_visualizer.h:129`. This makes sure, that each spline in the visualizer gets a different id, so that multiple splines can be printed at the same time.